src/ts/components/errormessageoverlay.ts

```diff
@@ -99,22 +101,25 @@ export class ErrorMessageOverlay extends Container<ErrorMessageOverlayConfig> {
     let config = <ErrorMessageOverlayConfig>this.getConfig();
 
     player.on(player.exports.Event.Error, (event: ErrorEvent) => {
-      let message = event.message;
+      let description = ErrorEventUtil.defaultErrorMessagesMapping[event.code] as string;
 
-      // Process message translations
+      let name = event.name;
+      let message = description + '\n(' + name + ')'; // default error message style
```

Comment:
The message should be, by priority:
 1. The output of an externally supplied translator
 2. The output if the internal default translator (`defaultErrorMessagesMapping`)
 3. `${error.code} ${error.name}`

The `ErrorMessageTranslator` shall not receive a `message`, only output one.